tests/test_project.py

```diff
@@ -72,16 +83,77 @@ def test_subpackage(baked_with_development_dependencies):
     if IS_WINDOWS_CI:
         # On Windows CI python and pip executable are in different paths
         bin_dir = ''
-    build_output = run([f'{bin_dir}python', 'setup.py', 'build'], project_dir)
-    assert build_output.returncode == 0
+    result = run([f'{bin_dir}python', 'setup.py', 'build'], project_dir)
```

Comment:
Can you add a comment to explain why build is used here?

Reply:
added comments